uberfire-extensions/uberfire-preferences-ui-client/src/main/java/org/uberfire/ext/preferences/client/central/form/DefaultPreferenceForm.java

```diff
@@ -113,11 +118,21 @@ private void createPropertiesEditorCategory() {
             final PropertyEditorFieldInfo fieldInfo = createFieldInfo(propertyName,
                                                                       propertyType,
                                                                       propertyValue);
-
+            if(property.getValue() == PropertyFormType.COMBO) {
```

Comment:
Missing space after `if`. There are other cases in this PR.

Reply:
I think it is best to move this logic into the `createFieldInfo` method and add a test for this case.

Reply:
Hello @paulovmr thanks for the feedback. I fixed the ifs and moved the logic into `createFieldInfo` method. For testing would you suggest create a property of enum type in `PortablePreferenceMock` or create a new preference? If I have to modify `PortablePreferenceMock` would you kindly suggest the preferable way to re-generate `PortablePreferenceMockBeanGeneratedImpl` and `PortablePreferenceMockPortableGeneratedImpl`? Thanks!